src/pages/Notes/Notes.css

```diff
@@ -117,3 +117,118 @@
 .notecard-bg-yellow {
   background-color: var(--notecard-bg-yellow);
 }
+
+@media screen and (max-width: 655px) {
+  .notes-container {
+    width: 85%;
+    padding: 2rem 1rem;
+  }
+
+  .add-note-inp {
+    width: 32rem;
+    margin-right: 3rem;
+    margin-left: 7rem;
+  }
+
```

Comment:
suggestion-here can have utility class repeated code.

Reply:
Sure I will do that